api-operator/pkg/config/system.go

```diff
@@ -43,3 +45,31 @@ func SetOperatorNamespace() {
 		OperatorNamespace = ns
 	}
 }
+
+// getWatchClusterLevel Checks whether operator is used cluster wide or namespace wide.
+func getWatchClusterLevel() (watchClusterLevel bool) {
+	watchClusterLevelValue, _ := os.LookupEnv(WatchClusterLevel)
+	watchClusterLevel, _ = strconv.ParseBool(watchClusterLevelValue)
+	return watchClusterLevel
+}
+
+// GetWatchNamespaces If cluster wide is enabled, return empty list or comma separated namespaces. If cluster wide is disabled,
+// used the operator deployed namespace.
+func GetWatchNamespaces() (watchNamespaces string) {
+
+	watchClusterLevel := getWatchClusterLevel()
+
+	if watchClusterLevel {
+		ns, _ := k8sutil.GetWatchNamespace()
+		if ns != "" {
+			watchNamespaces = ns
+		} else {
+			watchNamespaces = ""
+		}
+
+	} else {
+		watchNamespaces = OperatorNamespace
```

Comment:
shall we check `OperatorNamespace` and `SystemNamespace` here, and if not equals panic execution?